sidekiq_autoscale.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "addressable", "~> 2.8"
   s.add_dependency "k8s-ruby", ">= 0.12"
   s.add_dependency "platform-api", "~> 2.2"
-  s.add_dependency "railties", "= 5.2.6"
+  s.add_dependency "railties", ">= 5.2.6"
```

```suggestion
  s.add_dependency "railties", "~> 5.2"
```

Using the "pessimistic" (`~>`) operator here may be better. `~> 5.2` will depend on everything  from 5.2, up-to-but-excluding 6.0. Major version changes usually include breaking changes, so it's best to cross those boundaries explicitly.

---

The intention is specifically for >= 5.2 to include 6.  We need to be able to include this in our rspec against Rails 6.